lektor/devserver.py

```diff
@@ -53,6 +53,7 @@ def run(self):
                 for ts, _, _ in watcher:
                     if self.last_build is None or ts > self.last_build:
                         self.build()
+                        trigger_reload()
```

Comment:
> There are other paths which trigger a rebuild: hitting the "refresh build" button in the admin UI is one.

It seems not the case, it only fires a `clean/prune` action.

Reply:
> > There are other paths which trigger a rebuild: hitting the "refresh build" button in the admin UI is one.
> 
> It seems not the case, it only fires a `clean/prune` action.

Erm.  That appears to be a bug in the frontend, I think (maybe?) The reload button is poking the `/api/clean` which, as you say, just does a prune.  There is an `/api/build` which does a build_all, though.  (It appears to be invoked by the frontend as part of the publish process.)

In any case, are there any times build_all is called when you wouldn't want to trigger a reload?  If not, then build_all is the right place to call the trigger.

Reply:
I didn't hook into `Builder` class because I don't want it to affect other behaviors than dev server, where `BackgroundBuilder` seems the proper place to do it.